DgmlPowerTools.2022/DgmlPowerTools.2022/Utilities/ProjectDependencies.cs

```diff
@@ -14,7 +14,7 @@
 using CodeNodeCategories = Microsoft.VisualStudio.Progression.CodeSchema.NodeCategories;
 using CodeNodeProperties = Microsoft.VisualStudio.Progression.CodeSchema.Properties;
 using System.Diagnostics;
-using System.Threading.Tasks;
+using System.Windows.Media;
```

Comment:
What change required this new namespace?

Reply:
`nugetNode.SetStroke(Brushes.Red);`
SetStroke expects a Brush object which is in Windows.Media namespace.

Reply:
Right, makes sense, thanks.